xwiki-platform-core/xwiki-platform-notifications/xwiki-platform-notifications-ui/src/main/resources/XWiki/Notifications/MailTemplate.xml

```diff
@@ -137,7 +138,7 @@
     &lt;/tr&gt;&lt;/table&gt;
 
     ## Events div
-    &lt;div style="background:white;padding:1em; border-left: 1px solid #eaeaea;border-right: 1px solid #eaeaea;border-bottom: 1px solid #ddd;border-top: 1px solid #f0f0f0;"&gt;
+    &lt;div style="background:$!{theme.notificationEmailInnerBg};padding:1em; border-left: 1px solid #eaeaea;border-right: 1px solid #eaeaea;border-bottom: 1px solid #ddd;border-top: 1px solid #f0f0f0;"&gt;
```

Comment:
I would have reused some existing variables, if they made sense. 
Also instead of using the old .vm variables, you could have made the SSX to use LESS and use LESS @variables.